src/mame/machine/p2000t_mdcr.cpp

```diff
@@ -0,0 +1,323 @@
+// license:BSD-3-Clause
+// copyright-holders:Erwin Jansen
+/**********************************************************************
+
+    Philips P2000T Mini Digital Cassette Recorder  emulation
+
+**********************************************************************/
+
+#include "p2000t_mdcr.h"
+#include "emu.h"
+#include "formats/p2000t_cas.h"
```

Comment:
To make the build behave consistently with and without prefix headers enabled, this needs to be:
```
#include "emu.h"
#include "p2000t_mdcr.h"
```
(i.e. prefix first, then module header)

Reply:
Will fix